source/crud/tests/unified/updateMany-sort.yml

```diff
@@ -0,0 +1,56 @@
+description: updateMany-sort
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: client0
+      observeEvents:
+        - commandStartedEvent
+  - database:
+      id: database0
+      client: client0
+      databaseName: crud-tests
+  - collection:
+      id: collection0
+      database: database0
+      collectionName: coll0
+
+initialData:
+  - collectionName: coll0
+    databaseName: crud-tests
+    documents:
+      - { _id: 1, x: 11 }
+      - { _id: 2, x: 22 }
+      - { _id: 3, x: 33 }
+
+tests:
+  - description: UpdateMany with sort option unsupported
+    operations:
+      - name: updateMany
+        object: collection0
+        arguments:
+          filter: { _id: { $gt: 1 } }
+          sort: { _id: -1 }
```

Comment:
I'd be OK with removing this test entirely since it's not testing client-side behavior at all.

We should be able to trust that the server is testing that `multi: true` and `sort` conflict.

Reply:
I will remove this test if the Go driver is the only one who shares the `UpdateOptions` for both update_one and update_many. I will also separate `UpdateOneOptions` and `UpdateManyOptions` because the Go driver will introduce a new options interface anyway in v2.0.

Reply:
In the interest of pragmatism, I'm completely OK with keeping the singular UpdateOptions and leaving a note about `sort` not applying to `updateMany`.

Also, I think this was previously overlooked in [one of my previous comments](https://github.com/mongodb/specifications/pull/1644#discussion_r1739499229), but shouldn't this PR entail a change to [Write Models](https://github.com/mongodb/specifications/blob/master/source/crud/bulk-write.md#write-models) in the bulk write spec? That has a separate model class for UpdateOne, so I assume `sort` can just go there (no concern with a common options interface as we have in the CRUD spec).